hooks/queries/nft.ts

```diff
@@ -57,3 +61,40 @@ export const fetchNFTbyMint = (connection: Connection, pubkey: PublicKey) => {
     },
   })
 }
+
+export const useOwnerVerifiedCollectionsQuery = (owner: PublicKey) => {
+  const { connection } = useConnection()
+  const { data: ownedNfts } = useDigitalAssetsByOwner(owner)
+  const network = getNetworkFromEndpoint(connection.rpcEndpoint)
+  const verifiedNfts = filterAndMapVerifiedCollections(ownedNfts)
+
+  const enabled = owner !== undefined && ownedNfts !== undefined
+  const query = useQuery({
```

Comment:
I'm not sure it really makes sense for this to use useQuery because there's nothing to cache at the owner level. 

Here's what we want cached:
- what NFTs `owner` owns (thats cached by useDigitalAssetsByOwner)
- the collection nft data (that's cacehd by `fetchNFTbyMint`)

So there is nothing else to cache or key. Instead, you can just use `useAsync` for this and no key.

Reply:
And you could just put this in regular hooks folder, or even just in the one file it gets used in (I assume its just one? the wizard?)

Reply:
modify useQuesry to useAync instead.